lib/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,36 @@
+import { PropsWithChildren } from 'react'
+import './styles.css' // Importing the dedicated CSS file
+
+export interface TimelineEventProps {
+  date: string
+  title: string
+  description: string
+}
+
+export const TimelineEvent = ({ date, title, description }: TimelineEventProps) => {
+  return (
+    <li className="timeline-list-item">
+      <span className="timeline-date">{date}</span>
+      <div className="timeline-content">
+        <h3 className="timeline-title">{title}</h3>
+        <p className="timeline-description">{description}</p>
+      </div>
+    </li>
+  )
+}
```

Comment:
This should be done in a subcomponent so we can use:

```
<Timeline>
    <Timeline.Item />
</Timeline>
```

This component would have it's own props for the content we are passing into it

Reply:
Done:
      <Timeline>
        {sampleEvents.map((event, index) => (
          <Timeline.Item key={index} date={event.date} title={event.title} description={event.description} />
        ))}
      </Timeline>